uips/uip-draft_spend_backreferences.md

```diff
@@ -0,0 +1,106 @@
+# UIP: Spend Backreferences
+
+| uip | To be assigned |
```

Comment:
I think this should be 0002 instead of "to be assigned"?

Reply:
yep I initially did that and then I saw in UIP 1 it says the number should be assigned by the UIP editor so backed it out - @plaidfinch would you be able to act as UIP editor for this spec? Cool to make this one 0004?

Reply:
Yes let's make this 0004